src/common/adapters/jira.js

```diff
@@ -30,8 +30,13 @@ const adapter = {
       const title = ticketPageTitle(issue);
       const type = normalizeType($text('#type-val', issue));
       return fn(null, [{ id, title, type }]);
+    } else if ($has('.ghx-columns .ghx-issue.ghx-selected', doc)) { // Board view
+      const issue = $find('.ghx-columns .ghx-issue.ghx-selected', doc);
+      const id = $attr('.ghx-key', issue, 'aria-label');
+      const title = $text('.ghx-summary', issue);
+      const type = normalizeType($attr('.ghx-field-icon', issue, 'data-tooltip'));
+      return fn(null, [{ id, title, type }]);
```

Comment:
I don't have a jira at hand to confirm this magic DOM fiddling. But you probably got it right